src/libdredd/src/mutate_visitor.cc

```diff
@@ -734,7 +692,6 @@ bool MutateVisitor::TraverseCompoundStmt(clang::CompoundStmt* compound_stmt) {
 }
 
 void MutateVisitor::AddMutation(std::unique_ptr<Mutation> mutation) {
-  UpdateStartLocationOfFirstFunctionInSourceFile();
```

Comment:
We introduced the AddMutation function due to the need to call UpdateStartLocation... before each addition of a mutation. I wondered therefore if we should revert this and get rid of the helper, and outline the mutation_tree_path.back()... statements. However, on balance I think it's nice to encapsulate the management of the mutation tree into this function so let's keep it. (So, no action - feel free to resolve, just FYI.)